### PR TITLE
Add setup instructions

### PR DESCRIPTION
**Pull request by nbalmoori**
Hey Jordan check out my code!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,5 @@ Wow we are really rocking this Nikki!
 git clone git@github.com:nbalmoori/git-callobaration-practice.git
 ```
 2. Check out our amazing README file
+
+3. HERE'S nikki's 3rd instruction
```
Comment (jordan-sullivan) [inline README.md:11]: Please capitalize your name, otherwise, looks great!

**Review by jordan-sullivan — CHANGES_REQUESTED** [review]
Please see single comment, thanks!

File: README.md (hunk 1 above)
Comment (nbalmoori) [reply README.md]: Hey - just fixed it!

**Review by jordan-sullivan — APPROVED** [review]
Perfect-o~

**Pull request merged**